dub.selections.json

```diff
@@ -1,7 +1,7 @@
 {
 	"fileVersion": 1,
 	"versions": {
-		"dub": "1.29.0",
+		"dub": {"repository":"git+https://github.com/dlang/dub.git","version":"ade359c4d8ea8e9cbd8fc5d11cd8ff654d3cb53c"},
```

Comment:
This is an issue.

Reply:
Sure, that's why it's a draft for now - waiting for dub v1.30 to be released.